### ISSUE TITLE
UI: Separate Resources Under 'Articles' in 'Introduction to Open Source' Page

### ISSUE THREAD

**Issue by Sriparno08** [Sriparno08/Openpedia#348]
## Category

- [ ] Documentation
- [ ] Resource Addition
- [ ] Codebase
- [x] User Interface
- [ ] Feature Request

## Description

We need to separate the resources under 'Articles' in 'Introduction to Open Source' page by making them exist in separate boxes. It can look like this:

<p align="center">
  <img src="https://github.com/Sriparno08/Openpedia/assets/89148144/3df0d084-9ec2-4723-a6a4-93abf4d39c10">
</p>

**Comment by Sneha123-zudo** [comment]
Please assign me this issue under JWOC'24

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once your PR for it is merged, you will be considered for this issue. If anyone asks to be assigned before your PR is merged, he/she will be assigned.

**Comment by Sneha123-zudo** [comment]
Please assign me this issue under JWOC'24

**Comment by Sriparno08** [comment]
You're assigned.
